src/main/kotlin/io/github/anki/anki/service/CardsService.kt

```diff
@@ -62,6 +70,19 @@ class CardsService(
         }
     }
 
+    private fun requestDeleteCard(cardId: String): Mono<Message> =
+        natsClient.request(
+            NatsSubject.Card.CARD_DELETE_COMMAND,
+            DeleteCardRequest.newBuilder().setCardId(cardId).build().toByteArray(),
+        )
+            .toMono()
+
+    private fun validateDeleteCardResponse(msg: Message): Mono<Unit> {
```

Comment:
You need not return mono or flux on functions that don't use some out calls like with db.